lib/acx/horde/node_observer.ex

```diff
@@ -0,0 +1,38 @@
+defmodule Acx.Horde.NodeObserver do
```

Comment:
I think you would want this in your application not the external lib. You would still be able to alias Acx.Horde.HordeRegistry and Acx.EnforcerSupervisor in the your main application